### ISSUE TITLE
Feature Resquest:

### ISSUE THREAD

**Issue by ru4ert** [ipatalas/vscode-sprint-planner#45]
Sync tasks from Azure DevOps should fetch description too!

**Comment by ru4ert** [comment]
Current Sync does this:

![image](https://github.com/ipatalas/vscode-sprint-planner/assets/47078678/03ab3826-508c-4631-b979-9f09906e1f9c)

The publish was like this:

![image](https://github.com/ipatalas/vscode-sprint-planner/assets/47078678/301e55b8-91c6-487d-bb16-d7b0b90565a8)

So the expectation ist that the description should be pulled/synced too.

**Comment by ipatalas** [comment]
There was a reason this has never been implemented. Azure DevOps don't have a concept of plain text descriptions. Even if you do a simple thing like this:
![image](https://github.com/ipatalas/vscode-sprint-planner/assets/2623329/178b11fe-f356-462d-9a11-f7598514610e)
The API will return this:
```json
"System.Description": "<div>Test description&nbsp;<span>&#128522;</span> </div><div>Second line </div>",
```

Even if I do a simple parsing (each <div> being separate line) we might lose some of the original formatting and that's what I was afraid of. If you have good idea on how to handle that please let me know.

**Comment by ipatalas** [comment]
> Render it as HTML? 

Yeah, that's the only reasonable option to avoid losing anything but then it becomes a problem for non-technical people who don't know HTML. I was surprised but also roles like PM use this extension and they don't necessarily have technical background. That being said I'll play around with it and maybe introduce this as opt-in experimental feature so users have to explicitly enable it first and accept the risk :)

> 
> So, then its possible do edit the description as html. Does VSCode provides some sort of RichText Editor which is able to render html?

Yes, there is a way to embed HTML within any other language. I don't know how to do that yet but I'll figure it out. If it's too much work then still HTML can be just not highlighted at least in first version of this feature.
